distributed/deploy/ssh.py

```diff
@@ -58,6 +58,8 @@ class Worker(Process):
         The python module to run to start the worker.
     connect_options: dict
         kwargs to be passed to asyncssh connections
+    remote_python: str
+        Path to Python on remote node to run this worker.
```

Comment:
I like that you've chosen to be consistent with the previous value in dask-ssh.  I'm curious, without that constraint do you think that there is a better name here?

Reply:
I like `remote_python` ... also `python_path` can be an alternative.